source/taggedalgebraic/meson.build

```diff
@@ -3,6 +3,7 @@ taggedalgebraic_src = [
 	'taggedalgebraic.d',
 	'taggedunion.d',
 	'visit.d',
+	'package.d'
```

Comment:
Package.d now appears twice in the source list, since I've added it already in my previous merge request :)